config/routes.rb

```diff
@@ -7,14 +7,15 @@
 
   get 'artists/:id' => 'artists#show', as: :artistid
   get '/artists/', to: 'artists#all_artists'
-
   get 'buyers/:id' => 'buyers#show', as: :buyerid
+  get 'artists/works/:id' => 'artists#works'
 
   namespace :api, defaults: { format: :json } do
     resources :works, :only => [:show, :create, :update, :destroy]
     resources :artists, :only => [:index, :show, :create, :update, :destroy]
     resources :requests, :only => [:show, :create, :update]
     get 'artists/works/:id' => 'artists#works'
+    post 'commissions/' => 'commissions#create'
```

Comment:
we should structure commissions the same way that we structure works, artists, and requests above (i.e. `resources :requests, :only...`